index.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] Kindly update the HTML file as stated in the project requirement ⬇️ 
![image](https://github.com/Drew246/firstMicroProject/assets/68823331/45fd5d8a-137d-485b-960d-123755e34708)

Reply:
fixed